### ISSUE TITLE
Navbar issue on using dark theme

### ISSUE THREAD

**Issue by psankhe28** [dromara/hertzbeat#169]
**Is your feature request related to a problem? Please describe.**
When the dark theme is used, the navbar link of the page on which we are present appears in purple shade and it isn't clearly visible
**Describe the solution you'd like**

**Describe alternatives you've considered**
Changing its colour to white

**Comment by tomsun28** [comment]
👍👍thanks for this issue, we will fix it later

**Comment by psankhe28** [comment]
> 👍👍thanks for this issue, we will fix it later

Is this project available for applying in OSPP?

**Comment by tomsun28** [comment]
> Is this project available for applying in OSPP?
yes, there is a project [Hertzbeat official website UI revision and document i18n] relate it, you can contact to the project mentor @shimingxy , but the apply deadline is tonight, It seems that there are already student matching.
